testing/sdk_repository.py

```diff
@@ -96,18 +100,194 @@ def remove_universe_repos(stub_urls):
     log.info('Finished removing universe repos')
 
 
-def universe_session():
+def add_package_registry_stub() -> dict:
+    stub_url = os.environ['PACKAGE_REGISTRY_STUB_URL']
+    with urllib.request.urlopen(stub_url) as url:
+        repo = json.loads(url.read().decode())
+        min_supported = repo["packages"][0]["minDcosReleaseVersion"]
+
+    version = sdk_cmd.dcos_version()
```

Comment:
Is there any reason why we could we not just call `sdk_utils.dcos_version_at_least(min_supported)`?